### PR TITLE
Updating default value of max list items per call

### PR DESCRIPTION
**Pull request by singhravidutt**

### CODE REVIEW

**Review by medb — CHANGES_REQUESTED** [review]
Please update default value in the `gcs/CONFIGURATION.md` file and add a release note to the `gcs/CHANGES.md` file.

**Comment by singhravidutt** [comment]
@medb updated `gcs/CONFIGURATION.md` and `gcs/CHANGES.md`.

File: gcs/CHANGES.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
 ### 3.0.0 - 2022-XX-XX
 
-1.  Remove Hadoop 2.x support.
+1. Remove Hadoop 2.x support.
 
-1.  Update all dependencies to the latest versions.
+2. Update all dependencies to the latest versions.
```
Comment (medb) [inline gcs/CHANGES.md:5]: I think previous formatting was better (this is a default google3 md format: http://go/mdformat#intellij). It allows, for example, re-arrange release notes without changing their numbers manually.

**Review by medb — CHANGES_REQUESTED** [review]

File: gcs/CHANGES.md [hunk 2]
```diff
@@ -35,42 +35,49 @@
...
+     ```
+
+14. Updated default value for max item to return from list Cloud Storage
+    requests via following property:
 
     ```properties
-    fs.gs.outputstream.type
-    ```
+     fs.gs.list.max.items.per.call
+     ```
```
Comment (medb) [inline gcs/CHANGES.md:80]: We usually document default values changes like:

```md
1.  Set default value for `fs.gs.list.max.items.per.call` property to `5000`.
```

**Review by medb — CHANGES_REQUESTED** [review]

File: gcs/CHANGES.md (hunk 2 above)
Comment (singhravidutt) [reply gcs/CHANGES.md]: Reverted.

File: gcs/CHANGES.md (hunk 1 above)
Comment (singhravidutt) [reply gcs/CHANGES.md]: Reverted.

**Review by abmodi — APPROVED** [review]

**Review by medb — APPROVED** [review]

**Pull request merged**